java/spacewalk-java.spec

```diff
@@ -634,9 +634,13 @@ rm -rf $RPM_BUILD_ROOT/classes/com/redhat/rhn/common/conf/test/conf
 rm -rf $RPM_BUILD_ROOT%{_datadir}/rhn/unittest.xml
 %endif
 
+# Prettifying symlinks, excluding openSUSE
```

Comment:
And the reason is... at Uyuni we still use "our" `jboss-logging` package (will get rid of it with the migration to Leap 15.3: https://github.com/SUSE/spacewalk/issues/14677)

Reply:
Exactly, do you want me to mention the reason in the comment?

Reply:
No, don't worry, it's fine as it is :-)